content/contributing/documentation/content_guidelines.rst

```diff
@@ -2,228 +2,278 @@
-============================
+   The following fields are available on the :guilabel:`Replenishment` report:
 
-When writing documentation about a given topic, try to keep pages within the same folder organized.
+   - :guilabel:`Product`: the product that requires a replenishment
+   - :guilabel:`Location`: the specific location where the product is stored
+   - :guilabel:`Warehouse`: the warehouse where the product is stored
+   - :guilabel:`On Hand`: the amount of product currently available
 
-For most topics, a single page should do the job. Place it in the appropriate section of the
-documentation (e.g., content related to the CRM app go under :menuselection:`Applications
--> Sales -> CRM`) and follow the :ref:`document structure <contributing/document-structure>`
-guidelines.
+   **Numbered list**
 
-For more complex topics, you may need several pages to cover all their aspects. Usually, you will
-find yourself adding documentation to a topic that is already partially covered. In that case,
-either create a new page and place it at the same level as other related pages or add new sections
-to an existing page. If you are documenting a complex topic from scratch, organize your content
-between one parent page (the :abbr:`TOC (Tree Of Contents)` page) and several child pages. Whenever
-possible, write content on the parent page and not only on the child pages. Make the parent page
-accessible from the navigation menu by using the :ref:`show-content
-<contributing/document-metadata>` metadata directive.
+   To create a new website page, proceed as follows:
+
+   #. - Either open the **Website** app, click :guilabel:`+ New` in the top-right corner, then
+        select :guilabel:`Page`;
+      - Or go to :menuselection:`Website --> Site --> Pages` and click :guilabel:`New`.
 
-.. _contributing/content-images:
+   #. Enter a :guilabel:`Page Title`; this title is used in the menu and the page's URL.
+   #. Click :guilabel:`Create`.
+   #. Customize the page's content and appearance using the website builder, then click
+      :guilabel:`Save`.
+
+.. seealso::
+   :ref:`RST cheat sheet: lists <contributing/rst/lists>`
+
+.. _contributing/content/images:
 
 Images
 ======
 
-Adding a few images to illustrate your text helps the readers to understand and memorize your
-content. However, avoid adding too many images: it isn't necessary to illustrate all steps and
-features, and it may overload your page.
+Adding a few images to illustrate text helps the readers understand and memorize the content.
+However, images should never replace text: written instructions should be complete and clear on
+their own, without relying on visual aids. Use images sparingly, for example, to highlight a
+particular point or clarify an example.
 
 .. important::
-   Don't forget to :ref:`compress your PNG files with pngquant
+   Do not forget to :ref:`compress your PNG files with pngquant
    <contributing/documentation/first-contribution>`.
```

Comment:
I'm wondering since we don't have a full use + installation guide if it would be more useful to just directly link right to the [pngquant site](https://pngquant.org/)? When I clicked the internal link it brought me to the anchor on the other page, yes, however that list item is only a few words long and easy to miss among the larger list of other steps...wdyt?